src/Engine/ProtoCore/Parser/AssociativeAST.cs

```diff
@@ -1287,6 +1289,26 @@ public MethodAttributes(bool hiddenInLibrary = false, string msg = "")
         }
     }
 
+    public class ExternalAttributes
```

Comment:
Ok perhaps a future enhancement task can be added reflect this. Not sure if youve already done it.